src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -4137,10 +4137,7 @@ public void Nullable()
             Assert.True(parsedArgs.CompilationOptions.Nullable);
 
             parsedArgs = DefaultParse(new[] { @"/nullable-", "a.cs" }, WorkingDirectory);
-            parsedArgs.Errors.Verify(
-                // error CS8630: Invalid 'nullable' value: 'False' for C# 7.0. Please use language version 8.0 or greater.
```

Comment:
This doesn't feel right. The user is using C# 7, they should get a warning if they explicitly try and control the nullable features.

Reply:
> This doesn't feel right. The user is using C# 7, they should get a warning if they explicitly try and control the nullable features.

This scenario is turning feature off. I don't think there should be any diagnostics, regardless of language version.

---
In reply to: [237254749](https://github.com/dotnet/roslyn/pull/31420#discussion_r237254749) [](ancestors = 237254749)